packages/material-ui-unstyled/package.json

```
@@ -3,13 +3,14 @@
   "version": "5.0.0-alpha.21",
   "private": false,
   "author": "Material-UI Team",
-  "description": "Material-UI Unstyled - collection of unstyled React components.",
+  "description": "Accessible unstyled React components.",
```

Can we remove this? The term is used inflationary and lost its meaning a while ago. Instead of contributing to making it meaningless we can be confident enough to omit it.

---

"this" are you referring to "Accessible"?

```suggestion
  "description": "Unstyled React components.",
```